### ISSUE TITLE
1.4.3 - System / Info - External Module = Unknown

### ISSUE THREAD

**Issue by Target0815** [FrSkyRC/ETHOS-Feedback-Community#2095]
Hi,

under System / Info, Unknown is always displayed for External Module:

![ethos_unknown_module_1](https://user-images.githubusercontent.com/31627548/202747459-0c9275e2-e688-4ada-87bb-5e03e83603ed.jpg)

Although a Crossfire module is installed for the current model, for example:

![ethos_unknown_module_2](https://user-images.githubusercontent.com/31627548/202747684-b672cab3-a517-40ae-8ab9-702553214f8a.jpg)

As info it would be sufficient if the type is setup under RF System at System / Info.

**Comment by bsongis-frsky** [comment]
External Module version will be remove from there.

**Comment by Target0815** [comment]
I find the information in the info screen quite useful, especially for support purposes. If the external module would not only say "Crossfire" or "ELRS", but "Crossfire V 6.17" or "ELRS V 3.0.1 HM ES24TX", that would be an important information. Of course, the same applies to MPM or Twin module.

**Comment by r1000ru** [comment]
Hello, @Target0815 . I agree with @bsongis-frsky : no this info - no issue)
Seriously, there are scenarios where it is impossible to determine the module type:
1. Selected model with internal RF - external module is not powered
2. Selected model with external protocol, different from external module support (selected multiprotocol, installed elrs)
3. Not installed LUA-script, that activate expernal module protocol.

**Comment by Target0815** [comment]
Yes, these problems could certainly exist. Only, if the info cannot be read out up to a timeout, it still says "unknown". And if it does, then just the info.

In one of the other issues here is a screenshot with the info from the Twin Pro module. So it is possible and you should not give up so quickly.

**Comment by bsongis-frsky** [comment]
External Module information kept only if it is a FrSky module
